tests/IResearch/ExpressionFilterTest.cpp

```diff
@@ -187,11 +187,9 @@ struct custom_sort : public irs::sort {
                                     filter_node_attrs, document_attrs, boost);
       }
 
-      return {
-          std::make_unique<custom_sort::prepared::scorer>(
-              sort_, segment_reader, term_reader, filter_node_attrs,
-              document_attrs),
-          [](irs::score_ctx* ctx, irs::score_t* res) {
+      return irs::ScoreFunction::Make<custom_sort::prepared::scorer>(
+
+          [](irs::score_ctx* ctx, irs::score_t* res) noexcept {
```

Comment:
Is it noexcept? Oh I understand it's noexcept because this requirement was missed in old signature